packages/core/app-layout/src/components/aboutSection/AppAboutSection.cy.ts

```diff
@@ -80,10 +80,10 @@ describe('<AppAboutSection />', () => {
 
     cy.mount(AppAboutSection, {
       props: {
-        title,
         description,
       },
       slots: {
+        title,
```

Comment:
Rather than changing the test for a slot, can we leave this test as-is and write a new test that covers the slot?

Reply:
Updated